### DISCUSSION TITLE
Provider Questions

### DISCUSSION THREAD

**Discussion by rahaaatul** [Goldenfreddy0703/Otaku#282] [Q&A]
- I thought Animix was dead, then what source is otaku using?
- Can you add providers from th[i](https://everythingmoe.com/#section-streaming)s list in Otaku? and an option to change `TLD` manually like `.com`/`.net` etc. for sites?
- H!Anime sources are not showing up, but when I go to the website both hd-1 and hd-2 sources are playing.

**Comment by rahaaatul** [comment]
Also check Kuro[i](https://kuroiru.co/app)ru out. All in one place for streaming links of every anime.

**Comment by Goldenfreddy0703** [comment]
1. It is but we find clones
2. No, cause we have to figure out how to map the streams and get the videos to play
3. They are new thanks to this new Otaku Update
4. Kuro.ru is probably dead lol
